notebooks/02.00-Introduction-to-NumPy.ipynb

```diff
@@ -2,56 +2,55 @@
  "cells": [
```

Comment:
<p>Testing regression</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Test comment to verify <strong>email</strong> notification.</p>

Reply:
<p>afdsfd</p>

Reply:
<p>a</p>